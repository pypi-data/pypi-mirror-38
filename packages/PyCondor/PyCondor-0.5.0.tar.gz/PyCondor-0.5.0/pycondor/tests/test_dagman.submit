JOB examplejob_20180316_02_arg_0 condor/submit/examplejob_20180316_02.submit
VARS examplejob_20180316_02_arg_0 ARGS="--length 50"
JOB examplejob_20180316_02_arg_1 condor/submit/examplejob_20180316_02.submit
VARS examplejob_20180316_02_arg_1 ARGS="--length 100"
JOB examplejob_20180316_02_arg_2 condor/submit/examplejob_20180316_02.submit
VARS examplejob_20180316_02_arg_2 ARGS="--length 200"

#Inter-job dependencies