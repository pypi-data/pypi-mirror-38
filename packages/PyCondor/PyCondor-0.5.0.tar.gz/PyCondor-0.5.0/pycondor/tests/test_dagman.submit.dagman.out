03/16/18 23:24:07 Setting maximum file descriptors to 80000.
03/16/18 23:24:07 ******************************************************
03/16/18 23:24:07 ** condor_scheduniv_exec.304385022.0 (CONDOR_DAGMAN) STARTING UP
03/16/18 23:24:07 ** /usr/bin/condor_dagman
03/16/18 23:24:07 ** SubsystemInfo: name=DAGMAN type=DAGMAN(10) class=DAEMON(1)
03/16/18 23:24:07 ** Configuration: subsystem:DAGMAN local:<NONE> class:DAEMON
03/16/18 23:24:07 ** $CondorVersion: 8.7.5 Nov 14 2017 BuildID: 424079 $
03/16/18 23:24:07 ** $CondorPlatform: x86_64_RedHat6 $
03/16/18 23:24:07 ** PID = 624
03/16/18 23:24:07 ** Log last touched time unavailable (No such file or directory)
03/16/18 23:24:07 ******************************************************
03/16/18 23:24:07 Using config source: /etc/condor/condor_config
03/16/18 23:24:07 Using local config sources: 
03/16/18 23:24:07    /etc/condor/config.d/00_logging
03/16/18 23:24:07    /etc/condor/config.d/00_security
03/16/18 23:24:07    /etc/condor/config.d/20_ganglia
03/16/18 23:24:07    /etc/condor/config.d/20_master_daemons
03/16/18 23:24:07    /etc/condor/config.d/20_schedd
03/16/18 23:24:07    /etc/condor/config.d/20_schedd_limit_num_jobs
03/16/18 23:24:07    /etc/condor/config.d/20_schedd_stats
03/16/18 23:24:07    /etc/condor/config.d/20_schedd_stats_rss
03/16/18 23:24:07    /etc/condor/config.d/20_shared_port
03/16/18 23:24:07    /etc/condor/config.d/20_submit
03/16/18 23:24:07    /etc/condor/config.d/60_npx_submit
03/16/18 23:24:07    /etc/condor/config.d/60_npx_submit_sane_reqs
03/16/18 23:24:07    /etc/condor/config.d/99_block_submissions_group_long
03/16/18 23:24:07    /etc/condor/condor_config.local
03/16/18 23:24:07 config Macros = 132, Sorted = 132, StringBytes = 5358, TablesBytes = 4904
03/16/18 23:24:07 CLASSAD_CACHING is ENABLED
03/16/18 23:24:07 Daemon Log is logging: D_ALWAYS D_ERROR
03/16/18 23:24:07 DaemonCore: No command port requested.
03/16/18 23:24:07 DAGMAN_USE_STRICT setting: 1
03/16/18 23:24:07 DAGMAN_VERBOSITY setting: 3
03/16/18 23:24:07 DAGMAN_DEBUG_CACHE_SIZE setting: 5242880
03/16/18 23:24:07 DAGMAN_DEBUG_CACHE_ENABLE setting: False
03/16/18 23:24:07 DAGMAN_SUBMIT_DELAY setting: 1
03/16/18 23:24:07 DAGMAN_MAX_SUBMIT_ATTEMPTS setting: 6
03/16/18 23:24:07 DAGMAN_STARTUP_CYCLE_DETECT setting: False
03/16/18 23:24:07 DAGMAN_MAX_SUBMITS_PER_INTERVAL setting: 5
03/16/18 23:24:07 DAGMAN_AGGRESSIVE_SUBMIT setting: False
03/16/18 23:24:07 DAGMAN_USER_LOG_SCAN_INTERVAL setting: 5
03/16/18 23:24:07 DAGMAN_DEFAULT_PRIORITY setting: 0
03/16/18 23:24:07 DAGMAN_SUPPRESS_NOTIFICATION setting: True
03/16/18 23:24:07 allow_events (DAGMAN_ALLOW_EVENTS) setting: 114
03/16/18 23:24:07 DAGMAN_RETRY_SUBMIT_FIRST setting: True
03/16/18 23:24:07 DAGMAN_RETRY_NODE_FIRST setting: False
03/16/18 23:24:07 DAGMAN_MAX_JOBS_IDLE setting: 1000
03/16/18 23:24:07 DAGMAN_MAX_JOBS_SUBMITTED setting: 0
03/16/18 23:24:07 DAGMAN_MAX_PRE_SCRIPTS setting: 20
03/16/18 23:24:07 DAGMAN_MAX_POST_SCRIPTS setting: 20
03/16/18 23:24:07 DAGMAN_MUNGE_NODE_NAMES setting: True
03/16/18 23:24:07 DAGMAN_PROHIBIT_MULTI_JOBS setting: False
03/16/18 23:24:07 DAGMAN_SUBMIT_DEPTH_FIRST setting: False
03/16/18 23:24:07 DAGMAN_ALWAYS_RUN_POST setting: False
03/16/18 23:24:07 DAGMAN_ABORT_DUPLICATES setting: True
03/16/18 23:24:07 DAGMAN_ABORT_ON_SCARY_SUBMIT setting: True
03/16/18 23:24:07 DAGMAN_PENDING_REPORT_INTERVAL setting: 600
03/16/18 23:24:07 DAGMAN_AUTO_RESCUE setting: True
03/16/18 23:24:07 DAGMAN_MAX_RESCUE_NUM setting: 100
03/16/18 23:24:07 DAGMAN_WRITE_PARTIAL_RESCUE setting: True
03/16/18 23:24:07 DAGMAN_DEFAULT_NODE_LOG setting: @(DAG_DIR)/@(DAG_FILE).nodes.log
03/16/18 23:24:07 DAGMAN_GENERATE_SUBDAG_SUBMITS setting: True
03/16/18 23:24:07 DAGMAN_MAX_JOB_HOLDS setting: 100
03/16/18 23:24:07 DAGMAN_HOLD_CLAIM_TIME setting: 20
03/16/18 23:24:07 ALL_DEBUG setting: 
03/16/18 23:24:07 DAGMAN_DEBUG setting: 
03/16/18 23:24:07 DAGMAN_SUPPRESS_JOB_LOGS setting: False
03/16/18 23:24:07 DAGMAN_REMOVE_NODE_JOBS setting: True
03/16/18 23:24:07 argv[0] == "condor_scheduniv_exec.304385022.0"
03/16/18 23:24:07 argv[1] == "-Lockfile"
03/16/18 23:24:07 argv[2] == "condor/submit/exampledagman_20180316_02.submit.lock"
03/16/18 23:24:07 argv[3] == "-AutoRescue"
03/16/18 23:24:07 argv[4] == "1"
03/16/18 23:24:07 argv[5] == "-DoRescueFrom"
03/16/18 23:24:07 argv[6] == "0"
03/16/18 23:24:07 argv[7] == "-Dag"
03/16/18 23:24:07 argv[8] == "condor/submit/exampledagman_20180316_02.submit"
03/16/18 23:24:07 argv[9] == "-MaxJobs"
03/16/18 23:24:07 argv[10] == "3000"
03/16/18 23:24:07 argv[11] == "-Suppress_notification"
03/16/18 23:24:07 argv[12] == "-CsdVersion"
03/16/18 23:24:07 argv[13] == "$CondorVersion: 8.7.5 Nov 14 2017 BuildID: 424079 $"
03/16/18 23:24:07 argv[14] == "-Dagman"
03/16/18 23:24:07 argv[15] == "/usr/bin/condor_dagman"
03/16/18 23:24:07 Workflow batch-name: <exampledagman_20180316_02.submit+304385022>
03/16/18 23:24:07 Workflow accounting_group: <>
03/16/18 23:24:07 Workflow accounting_group_user: <>
03/16/18 23:24:07 Warning: failed to get attribute DAGNodeName
03/16/18 23:24:07 DAGMAN_LOG_ON_NFS_IS_ERROR setting: False
03/16/18 23:24:07 Default node log file is: </home/jbourbeau/software/pycondor/examples/condor/submit/exampledagman_20180316_02.submit.nodes.log>
03/16/18 23:24:07 DAG Lockfile will be written to condor/submit/exampledagman_20180316_02.submit.lock
03/16/18 23:24:07 DAG Input file is condor/submit/exampledagman_20180316_02.submit
03/16/18 23:24:07 Parsing 1 dagfiles
03/16/18 23:24:07 Parsing condor/submit/exampledagman_20180316_02.submit ...
03/16/18 23:24:07 Dag contains 3 total jobs
03/16/18 23:24:07 Sleeping for 3 seconds to ensure ProcessId uniqueness
03/16/18 23:24:10 Bootstrapping...
03/16/18 23:24:10 Number of pre-completed nodes: 0
03/16/18 23:24:10 MultiLogFiles: truncating log file /home/jbourbeau/software/pycondor/examples/condor/submit/exampledagman_20180316_02.submit.nodes.log
03/16/18 23:24:10 DAG status: 0 (DAG_STATUS_OK)
03/16/18 23:24:10 Of 3 nodes total:
03/16/18 23:24:10  Done     Pre   Queued    Post   Ready   Un-Ready   Failed
03/16/18 23:24:10   ===     ===      ===     ===     ===        ===      ===
03/16/18 23:24:10     0       0        0       0       3          0        0
03/16/18 23:24:10 0 job proc(s) currently held
03/16/18 23:24:10 Registering condor_event_timer...
03/16/18 23:24:11 Sleeping for 1 s (DAGMAN_SUBMIT_DELAY) to throttle submissions...
03/16/18 23:24:12 Submitting HTCondor Node examplejob_20180316_02_arg_0 job(s)...
03/16/18 23:24:12 Adding a DAGMan workflow log /home/jbourbeau/software/pycondor/examples/condor/submit/exampledagman_20180316_02.submit.nodes.log
03/16/18 23:24:12 Masking the events recorded in the DAGMAN workflow log
03/16/18 23:24:12 Mask for workflow log is 0,1,2,4,5,7,9,10,11,12,13,16,17,24,27,35,36
03/16/18 23:24:12 submitting: /usr/bin/condor_submit -a dag_node_name' '=' 'examplejob_20180316_02_arg_0 -a +DAGManJobId' '=' '304385022 -a DAGManJobId' '=' '304385022 -batch-name exampledagman_20180316_02.submit+304385022 -a submit_event_notes' '=' 'DAG' 'Node:' 'examplejob_20180316_02_arg_0 -a dagman_log' '=' '/home/jbourbeau/software/pycondor/examples/condor/submit/exampledagman_20180316_02.submit.nodes.log -a +DAGManNodesMask' '=' '"0,1,2,4,5,7,9,10,11,12,13,16,17,24,27,35,36" -a ARGS' '=' '--length' '50 -a DAG_STATUS' '=' '0 -a FAILED_COUNT' '=' '0 -a notification' '=' 'never -a +DAGParentNodeNames' '=' '"" condor/submit/examplejob_20180316_02.submit
03/16/18 23:24:13 From submit: Submitting job(s).
03/16/18 23:24:13 From submit: 1 job(s) submitted to cluster 304385150.
03/16/18 23:24:13 	assigned HTCondor ID (304385150.0.0)
03/16/18 23:24:13 Sleeping for 1 s (DAGMAN_SUBMIT_DELAY) to throttle submissions...
03/16/18 23:24:14 Submitting HTCondor Node examplejob_20180316_02_arg_1 job(s)...
03/16/18 23:24:14 Adding a DAGMan workflow log /home/jbourbeau/software/pycondor/examples/condor/submit/exampledagman_20180316_02.submit.nodes.log
03/16/18 23:24:14 Masking the events recorded in the DAGMAN workflow log
03/16/18 23:24:14 Mask for workflow log is 0,1,2,4,5,7,9,10,11,12,13,16,17,24,27,35,36
03/16/18 23:24:14 submitting: /usr/bin/condor_submit -a dag_node_name' '=' 'examplejob_20180316_02_arg_1 -a +DAGManJobId' '=' '304385022 -a DAGManJobId' '=' '304385022 -batch-name exampledagman_20180316_02.submit+304385022 -a submit_event_notes' '=' 'DAG' 'Node:' 'examplejob_20180316_02_arg_1 -a dagman_log' '=' '/home/jbourbeau/software/pycondor/examples/condor/submit/exampledagman_20180316_02.submit.nodes.log -a +DAGManNodesMask' '=' '"0,1,2,4,5,7,9,10,11,12,13,16,17,24,27,35,36" -a ARGS' '=' '--length' '100 -a DAG_STATUS' '=' '0 -a FAILED_COUNT' '=' '0 -a notification' '=' 'never -a +DAGParentNodeNames' '=' '"" condor/submit/examplejob_20180316_02.submit
03/16/18 23:24:15 From submit: Submitting job(s).
03/16/18 23:24:15 From submit: 1 job(s) submitted to cluster 304385161.
03/16/18 23:24:15 	assigned HTCondor ID (304385161.0.0)
03/16/18 23:24:15 Sleeping for 1 s (DAGMAN_SUBMIT_DELAY) to throttle submissions...
03/16/18 23:24:16 Submitting HTCondor Node examplejob_20180316_02_arg_2 job(s)...
03/16/18 23:24:16 Adding a DAGMan workflow log /home/jbourbeau/software/pycondor/examples/condor/submit/exampledagman_20180316_02.submit.nodes.log
03/16/18 23:24:16 Masking the events recorded in the DAGMAN workflow log
03/16/18 23:24:16 Mask for workflow log is 0,1,2,4,5,7,9,10,11,12,13,16,17,24,27,35,36
03/16/18 23:24:16 submitting: /usr/bin/condor_submit -a dag_node_name' '=' 'examplejob_20180316_02_arg_2 -a +DAGManJobId' '=' '304385022 -a DAGManJobId' '=' '304385022 -batch-name exampledagman_20180316_02.submit+304385022 -a submit_event_notes' '=' 'DAG' 'Node:' 'examplejob_20180316_02_arg_2 -a dagman_log' '=' '/home/jbourbeau/software/pycondor/examples/condor/submit/exampledagman_20180316_02.submit.nodes.log -a +DAGManNodesMask' '=' '"0,1,2,4,5,7,9,10,11,12,13,16,17,24,27,35,36" -a ARGS' '=' '--length' '200 -a DAG_STATUS' '=' '0 -a FAILED_COUNT' '=' '0 -a notification' '=' 'never -a +DAGParentNodeNames' '=' '"" condor/submit/examplejob_20180316_02.submit
03/16/18 23:24:17 From submit: Submitting job(s).
03/16/18 23:24:17 From submit: 1 job(s) submitted to cluster 304385174.
03/16/18 23:24:17 	assigned HTCondor ID (304385174.0.0)
03/16/18 23:24:17 Just submitted 3 jobs this cycle...
03/16/18 23:24:17 Currently monitoring 1 HTCondor log file(s)
03/16/18 23:24:17 Reassigning the id of job examplejob_20180316_02_arg_0 from (304385150.0.0) to (304385150.0.0)
03/16/18 23:24:17 Event: ULOG_SUBMIT for HTCondor Node examplejob_20180316_02_arg_0 (304385150.0.0) {03/16/18 23:24:13}
03/16/18 23:24:17 Number of idle job procs: 1
03/16/18 23:24:17 Reassigning the id of job examplejob_20180316_02_arg_1 from (304385161.0.0) to (304385161.0.0)
03/16/18 23:24:17 Event: ULOG_SUBMIT for HTCondor Node examplejob_20180316_02_arg_1 (304385161.0.0) {03/16/18 23:24:15}
03/16/18 23:24:17 Number of idle job procs: 2
03/16/18 23:24:17 Reassigning the id of job examplejob_20180316_02_arg_2 from (304385174.0.0) to (304385174.0.0)
03/16/18 23:24:17 Event: ULOG_SUBMIT for HTCondor Node examplejob_20180316_02_arg_2 (304385174.0.0) {03/16/18 23:24:17}
03/16/18 23:24:17 Number of idle job procs: 3
03/16/18 23:24:17 DAG status: 0 (DAG_STATUS_OK)
03/16/18 23:24:17 Of 3 nodes total:
03/16/18 23:24:17  Done     Pre   Queued    Post   Ready   Un-Ready   Failed
03/16/18 23:24:17   ===     ===      ===     ===     ===        ===      ===
03/16/18 23:24:17     0       0        3       0       0          0        0
03/16/18 23:24:17 0 job proc(s) currently held
03/16/18 23:25:12 Currently monitoring 1 HTCondor log file(s)
03/16/18 23:25:12 Event: ULOG_EXECUTE for HTCondor Node examplejob_20180316_02_arg_0 (304385150.0.0) {03/16/18 23:25:09}
03/16/18 23:25:12 Number of idle job procs: 2
03/16/18 23:25:12 Event: ULOG_EXECUTE for HTCondor Node examplejob_20180316_02_arg_1 (304385161.0.0) {03/16/18 23:25:10}
03/16/18 23:25:12 Number of idle job procs: 1
03/16/18 23:25:12 Event: ULOG_EXECUTE for HTCondor Node examplejob_20180316_02_arg_2 (304385174.0.0) {03/16/18 23:25:10}
03/16/18 23:25:12 Number of idle job procs: 0
03/16/18 23:25:17 Currently monitoring 1 HTCondor log file(s)
03/16/18 23:25:17 Event: ULOG_JOB_TERMINATED for HTCondor Node examplejob_20180316_02_arg_0 (304385150.0.0) {03/16/18 23:25:12}
03/16/18 23:25:17 Number of idle job procs: 0
03/16/18 23:25:17 Node examplejob_20180316_02_arg_0 job proc (304385150.0.0) completed successfully.
03/16/18 23:25:17 Node examplejob_20180316_02_arg_0 job completed
03/16/18 23:25:17 DAG status: 0 (DAG_STATUS_OK)
03/16/18 23:25:17 Of 3 nodes total:
03/16/18 23:25:17  Done     Pre   Queued    Post   Ready   Un-Ready   Failed
03/16/18 23:25:17   ===     ===      ===     ===     ===        ===      ===
03/16/18 23:25:17     1       0        2       0       0          0        0
03/16/18 23:25:17 0 job proc(s) currently held
03/16/18 23:25:22 Currently monitoring 1 HTCondor log file(s)
03/16/18 23:25:22 Event: ULOG_JOB_TERMINATED for HTCondor Node examplejob_20180316_02_arg_2 (304385174.0.0) {03/16/18 23:25:20}
03/16/18 23:25:22 Number of idle job procs: 0
03/16/18 23:25:22 Node examplejob_20180316_02_arg_2 job proc (304385174.0.0) completed successfully.
03/16/18 23:25:22 Node examplejob_20180316_02_arg_2 job completed
03/16/18 23:25:22 Event: ULOG_JOB_TERMINATED for HTCondor Node examplejob_20180316_02_arg_1 (304385161.0.0) {03/16/18 23:25:20}
03/16/18 23:25:22 Number of idle job procs: 0
03/16/18 23:25:22 Node examplejob_20180316_02_arg_1 job proc (304385161.0.0) completed successfully.
03/16/18 23:25:22 Node examplejob_20180316_02_arg_1 job completed
03/16/18 23:25:22 DAG status: 0 (DAG_STATUS_OK)
03/16/18 23:25:22 Of 3 nodes total:
03/16/18 23:25:22  Done     Pre   Queued    Post   Ready   Un-Ready   Failed
03/16/18 23:25:22   ===     ===      ===     ===     ===        ===      ===
03/16/18 23:25:22     3       0        0       0       0          0        0
03/16/18 23:25:22 0 job proc(s) currently held
03/16/18 23:25:22 All jobs Completed!
03/16/18 23:25:22 Note: 0 total job deferrals because of -MaxJobs limit (3000)
03/16/18 23:25:22 Note: 0 total job deferrals because of -MaxIdle limit (1000)
03/16/18 23:25:22 Note: 0 total job deferrals because of node category throttles
03/16/18 23:25:22 Note: 0 total PRE script deferrals because of -MaxPre limit (20) or DEFER
03/16/18 23:25:22 Note: 0 total POST script deferrals because of -MaxPost limit (20) or DEFER
03/16/18 23:25:22 DAG status: 0 (DAG_STATUS_OK)
03/16/18 23:25:22 Of 3 nodes total:
03/16/18 23:25:22  Done     Pre   Queued    Post   Ready   Un-Ready   Failed
03/16/18 23:25:22   ===     ===      ===     ===     ===        ===      ===
03/16/18 23:25:22     3       0        0       0       0          0        0
03/16/18 23:25:22 0 job proc(s) currently held
03/16/18 23:25:22 Wrote metrics file condor/submit/exampledagman_20180316_02.submit.metrics.
03/16/18 23:25:22 Metrics not sent because of PEGASUS_METRICS or CONDOR_DEVELOPERS setting.
03/16/18 23:25:22 DAGMan Runtime Statistics:
EventCycleTimeStd = 1.493636648520801
SubmitCycleTimeAvg = 0.3699734040669033
LogProcessCycleTimeCount = 4.0
EventCycleTimeCount = 13.0
SubmitCycleTimeMin = 1.001358032226562E-05
EventCycleTimeMax = 5.404063940048218
SleepCycleTimeAvg = 5.004693764906663
EventCycleTimeSum = 5.796468019485474
SubmitCycleTimeStd = 1.383924099038633
LogProcessCycleTimeAvg = 0.001464247703552246
SleepCycleTimeMin = 5.00408411026001
LogProcessCycleTimeMin = 0.0001900196075439453
SleepCycleTimeCount = 13.0
SubmitCycleTimeMax = 5.178273916244507
SleepCycleTimeStd = 0.0003512451025833709
LogProcessCycleTimeStd = 0.002435160561227487
SubmitCycleTimeSum = 5.179627656936646
SubmitCycleTimeCount = 14.0
EventCycleTimeAvg = 0.4458821553450364
SleepCycleTimeMax = 5.00503396987915
EventCycleTimeMin = 2.503395080566406E-05
LogProcessCycleTimeMax = 0.005115985870361328
SleepCycleTimeSum = 65.06101894378662
LogProcessCycleTimeSum = 0.005856990814208984

03/16/18 23:25:22 **** condor_scheduniv_exec.304385022.0 (condor_DAGMAN) pid 624 EXITING WITH STATUS 0
